draft-ietf-quic-manageability.md

```diff
@@ -160,17 +162,24 @@ information may be exposed in QUIC packet headers:
   connection ID is implicit.
 
 - length: the length of the remaining quic packet after the length field,
-  present on
-  long headers. This field is used to implement coalesced packets during the
-  handshake (see {{coalesced}}).
+  present on long headers. This field is used to implement coalesced packets
+  during the handshake (see {{coalesced}}).
+
+- spin bit: the spin bit supports passive measurement of RTT, and is present on
+  QUIC short packet headers. See {{sec-rtt}}.
+
+Other information in packet headers is cryptographically obfuscated:
 
-- packet number: Every packet has an associated packet number; however, this
-  packet number is encrypted, and therefore not of use to on-path observers.
-  This packet number has a fixed location and length in long headers, and an
-  implicit location and encrypted variable length in short headers.
+- packet number: Most packets (with the exception of Version Negotiation and
+  Retry packets) have an associated packet number; however, this packet number
+  is encrypted, and therefore not of use to on-path observers. The offset of the
+  packet number is encoded in the header for packets with long headers, while it
+  is implicit (depending on Destination Connection ID length) in short header
+  packets. The length of the packet number is cryptographically obfuscated.
 
-- key phase: The Key Phase bit, present in short headers identifies the key
-  used to encrypt the packet during key rotation.
+- key phase: The Key Phase bit, present in short headers, specifies the keys
+  used to encrypt the packet, supporting key rotation. The Key Phase bit is
+  cryptographically obfuscated.
 
```

Comment:
Separately observe that Retry and Version Negotiation packets are not encrypted or obfuscated in any way.

Reply:
also done